test/tests/mesh/ghost_functors/tests

```diff
@@ -12,7 +12,7 @@
   [./geometric_edge_neighbor_two_2D]
     type = 'Exodiff'
     input = 'geometric_neighbors.i'
-    cli_args = 'Mesh/num_ghosted_layers=2 Outputs/file_base=geometric_edge_two_2D_out'
+    cli_args = 'UserObjects/ghost_uo/element_side_neighbor_layers=2 Outputs/file_base=geometric_edge_two_2D_out'
     exodiff = 'geometric_edge_two_2D_out.e'
```

Comment:
What is the exodiff actually testing for with regard to the number of ghosted layers in these tests?

Reply:
AuxVariable that contains the union of all processors ghosted elements.